index.html

```diff
@@ -0,0 +1,329 @@
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <section class="contact-form">
+        <article class="cover">
+           <h3 class="contact-header">Contact Me</h3>
+           <p class="contact-paragraph">If you have an application you are interested in developing,<br> 
+             a feature that you need built or a project that needs coding. <br>
+             I'd love to help with it.
+           </p>
+
+           <section class="design">
+              <div class="design-2">
+                <form action=" https://formspree.io/f/xyylppbj" method="POST">
+                <label>
+                    <input type="text" name="Names" class="names" id="name" placeholder="NSABIMANA Emmanuel" required="">
+                </label>
+
+                <label>
+                    <input type="email" name="E-mail" class="names" id="email" placeholder="emmy66418@gmail.com" required="">
+                </label>
+                  
+                <label>
+                  <textarea name="Message" class="message" placeholder="Write your message here!!" rows="5" id="message" required=""></textarea>
+                </label>
+                
+                  <input type="submit" value="Get It Touch" class="button">
+                </form>
+              </div>
+          </section>
+        </article>
+        
+      </section>
```

Comment:
- [ ] [Optional] It's better to stick to the design as much as possible (e.g., font, colors, images, tests, margins) using the templates in Figma you choose. 🖼️ 

![image](https://user-images.githubusercontent.com/24830039/124030906-f4f7e100-d9f6-11eb-9a91-ce08913db55d.png)